rover/src/main.c

```diff
@@ -257,16 +278,16 @@ uint32_t get_us(uint32_t trig, uint32_t echo, struct device *dev)
 	uint32_t stop_time;
 	uint32_t start_time;
 	gpio_pin_write(dev, trig, 1);
-	k_sleep(K_MSEC(10));
+	k_busy_wait(50);
```

Comment:
What is the performance benefit of using `k_busy_wait` over `k_sleep`? This will block the current thread for certain MS and no other thread will be scheduled until it lapses.